src/config/links.json

```diff
@@ -31,6 +31,7 @@
     "slack": "#4A154B",
     "snapchat": "#FFFC00",
     "spotify": "#1ED760",
+    "stackoverflow":"#F58025",
```

Comment:
Not sure this is very needed, but IMO, it would look better.
```suggestion
    "stackoverflow": "#F58025",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Inelegant

Reply:
yeah, updated it